### ISSUE TITLE
not working

### ISSUE THREAD

**Issue by qp1990** [salumguilherme/reportbot#3]
this script not working men, report to 2 people with rage and still stand

**Comment by jhilgenberg** [comment]
@qp1990 you need at least 11 Reports per Player to get them into OW Pool. i'm using almost the same script and its working very well.

**Comment by qp1990** [comment]
@jhilgenberg I have 50 acc..2 with rage to full and 0 banned.. Will it be because I'm using node 6.10?

Does the old reportbot work for you?

**Comment by salumguilherme** [comment]
Are you getting confirmation IDS when you report?

You should be getting the ok response on your console.

Sometimes I don't get a ban when I report but thats up to overwatch.

**Comment by qp1990** [comment]
@salumguilherme Yes, I put npm start and then enter id 64 inside the match. What version of the node are you using?

**Comment by maplinx** [comment]
not work, we report a rager (spinbot)
got confirmation report id
waited 12 hours, he still alive till now, no game ban
